fa/Universally Preferable Behaviour/p01_ch05_02_univ.md

```diff
@@ -121,22 +121,22 @@ To aid in swallowing this rather large conceptual pill, below is a table that he
 | **Example**                | Matter cannot be created or destroyed, merely converted to energy and back                        | If it is alive and warm-blooded, it is a mammal                                              | Stealing is wrong                                                                                                             |
 | **Hypothesis**             | Atoms share common structures and properties, and so behave in predictable and consistent manners | Organic matter has rules – or requirements – that are common across classifications          | Universally preferable behaviour shares common rules and requirements                                                         |
 | **Proof**                  | Logical consistency, empirical verification                                                       | Logical consistency, empirical verification                                                  | Logical consistency, empirical verification                                                                                   |
-| **Negative Proof Example** | If mass does not attract mass, theories relying on gravity are incorrect                          | If organisms do not naturally self-select for survival, the theory of evolution is incorrect | If communism succeeds relative to its stated goals, theories based on the universal validity of property rights are incorrect |
+| **Negative Proof Example** | If mass does not attract mass, theories relying on gravity are incorrect                          | If communism succeeds relative to its stated goals, theories based on the universal validity of property rights are incorrect |
```

Comment:
Hello, one column is missing in this table, I'll be re-adding it @gn-ahmad @romero350